rke/k8s_rke_versions.go

```diff
@@ -0,0 +1,89 @@
+package rke
+
+import "github.com/rancher/types/apis/management.cattle.io/v3"
+
+func loadRKEDefaultK8sVersions() map[string]string {
+	return map[string]string{
+		"0.2.3":   "v1.14.1-rancher1-2",
```

Comment:
@kinarashah same here. Can you describe how these keys are going to be accessed/used?